src/utils/string.ts

```diff
@@ -8,9 +8,30 @@ export const parsePageNumber = (value: any, defaultValue: number) => {
 export const startEndEllipsis = (value: string, endLength = 8, startLength = 16) => {
   if (value === undefined || value === null) return ''
   if (value.length <= startLength) return value
```

Comment:
if `endLength` is limited to >= 0, this branch is redundant.

Reply:
OK. I see. I will update later.

Reply:
@Keith-CY  Updated. https://github.com/nervosnetwork/ckb-explorer-frontend/pull/329/commits/bae0969dfc739d2ddebabc59e4eee36e39c24e62